src/scan.ts

```diff
@@ -15,18 +15,20 @@ import { checkImportedOperations } from "./comparison/compareOperations";
 import { importOperations } from "./input/importOperations";
 import { save } from "./actions/saveAnalysis";
 import { getArgs } from "./input/args";
+import { configuration } from "./configuration/settings";
 
-const VERSION = "0.3.6";
+const VERSION = "0.3.7";
```

Comment:
you could probably extract this from `package.json` to avoid duplication (or forgetting to update :p )

see https://github.com/LedgerHQ/vault-js/blob/065b45f8316c0fab6da01413658bd62619b512c2/packages/cli/src/cli.ts#L8-L9

Reply:
Great idea!